docs/yum-fs-snapshot.conf.5

```diff
@@ -21,6 +21,10 @@ utilizes configuration options in the form of
 This is a space delimited list of the mount points you do not wish to have
 snapshotted by this plugin.
 .SH OPTION
+.IP exclude_prefix
+This is a space delimited list of path prefixes to match mount points you
+do not want to have snapshotted by this plugin. (e.g. "/mnt")
```

Comment:
I would maybe add here an example of what having "/mnt" there would actually do (ie. basically the example in your commit message).